.github/workflows/_linux_accelerate.yml

```diff
@@ -0,0 +1,156 @@
+      python: ${{ inputs.python != '' && inputs.python || '3.10' }}
+      pytorch: ${{ inputs.pytorch != '' && inputs.pytorch || 'nightly' }}
+      PYTORCH_DEBUG_XPU_FALLBACK: 1
+      ZE_AFFINITY_MASK: 0
+      PARSE_JUNIT: ${{ github.workspace }}/torch-xpu-ops/.github/scripts/parse-junitxml.py
+    steps:
+      - name: Checkout torch-xpu-ops
+        uses: actions/checkout@v4
+        with:
+          path: torch-xpu-ops
+      - name: Checkout Accelerate
+        uses: actions/checkout@v4
+        with:
+          repository: huggingface/accelerate
+          ref: ${{ env.accelerate }}
+          path: accelerate
+      - name: Prepare Conda ENV
+        run: |
+          which conda && conda clean -ay
+          conda remove --all -y -n $CONDA_ENV_NAME || rm -rf $(dirname ${CONDA_EXE})/../envs/$CONDA_ENV_NAME
+          conda create -y -n $CONDA_ENV_NAME python=${{ env.python }}
+          source activate $CONDA_ENV_NAME
+          pip install junitparser
+      - name: Prepare Stock XPU Pytorch
+        run: |
+          source activate $CONDA_ENV_NAME
+          if [ -z "${{ inputs.nightly_whl }}" ]; then
+            pip install torch torchvision torchaudio --pre --index-url https://download.pytorch.org/whl/nightly/xpu
+          else
+            pip install torch==$(echo ${{ inputs.nightly_whl }}) torchvision torchaudio --pre --index-url https://download.pytorch.org/whl/nightly/xpu
+          fi
+      - name: Prepare Accelerate
+        run: |
+          source activate $CONDA_ENV_NAME
+          cd $WORK_DIR
+          pip install -e .
+          pip install -e ".[testing]"
+          rm -rf tests_log && mkdir -p tests_log
+          rm -rf reports
+          cp ${{ github.workspace }}/torch-xpu-ops/.github/scripts/spec.py ./
+      - name: Report installed versions
+        run: |
+          source activate $CONDA_ENV_NAME
+          echo "pip installed packages:"
+          pip list | tee ${{ github.workspace }}/$WORK_DIR/tests_log/pip_list.txt
+          echo "lspci gpu devices:"
+          lspci -d ::0380 | tee ${{ github.workspace }}/$WORK_DIR/tests_log/lspci_0380.txt
+          echo "GPU render nodes:"
+          cat /sys/class/drm/render*/device/device | tee ${{ github.workspace }}/$WORK_DIR/tests_log/device_IDs.txt
+          echo "xpu-smi output:"
+          xpu-smi discovery -y --json --dump -1
+      - name: Sanity check installed packages
+        run: |
+          source activate $CONDA_ENV_NAME
+          # These checks are to exit earlier if for any reason torch
+          # packages were reinstalled back to CUDA versions (not expected).
+          pip show torch | grep Version | grep xpu
+          pip show torchaudio | grep Version | grep xpu
+          pip show torchvision | grep Version | grep xpu
+          python -c 'import torch; exit(not torch.xpu.is_available())'
+      - name: Run -k backbone tests
+        env:
+          TEST_CASE: 'tests_backbone'
```

Comment:
Will it reuse transformers test case? Or it has itself tests?

Reply:
It has its own tests. I forgot to change this copying from Transformers workflow. Thanks for the catch. Fixed.